beacon_node/src/cli.rs

```diff
@@ -414,6 +414,35 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        will be used. Defaults to http://127.0.0.1:8545.")
                 .takes_value(true)
         )
+        .arg(
+            Arg::with_name("jwt-secrets")
+                .long("jwt-secrets")
+                .value_name("JWT-SECRETS")
+                .help("One or more comma-delimited file paths which contain the corresponding hex-encoded \
+                       JWT secrets for each execution endpoint provided in the --execution-endpoints flag. \
+                       The number of paths should be in the same order and strictly equal to the number \
+                       of execution endpoints provided.")
+                .takes_value(true)
+                .requires("execution-endpoints")
+        )
+        .arg(
+            Arg::with_name("jwt-id")
+                .long("jwt-id")
+                .value_name("JWT-ID")
+                .help("Used by the beacon node to communicate a unique identifier to execution nodes \
+                       during JWT authentication. It corresponds to the 'id' field in the JWT claims object.\
+                       Set to empty by deafult")
+                .takes_value(true)
+        )
+        .arg(
+            Arg::with_name("jwt-version")
+                .long("jwt-version")
+                .value_name("JWT-VERSION")
+                .help("Used by the beacon node to communicate a client version to execution nodes \
+                       during JWT authentication. It corresponds to the 'clv' field in the JWT claims object.\
+                       Set to the lighthouse version by deafult")
```

Comment:
I'm not sure that the Lighthouse version is used by default. I think we will just do `None` by default.

Reply:
Ohh yeah I meant to change it to default Lighthouse version but forgot. I'm not sure if it would be useful, happy to add it in though.